_delphi_utils_python/delphi_utils/validator/dynamic.py

```diff
@@ -263,7 +264,8 @@ def check_max_allowed_max_date(self, max_date, geo_type, signal_type, report):
                 ValidationFailure("check_max_max_date",
                                   geo_type=geo_type,
                                   signal=signal_type,
```

Comment:
does it make sense to add this here too?
```suggestion
                                  geo_type=geo_type,
                                  signal=signal_type,
                                  date=max_date,
```